### DISCUSSION TITLE
[Bug]: Money mask change value after load file with FileUpload

### DISCUSSION THREAD

**Discussion by ghinozzi** [leandrocfe/filament-ptbr-form-fields#55] [Q&A]
### What happened?

Ao carregar algum anexo no FileUpload o campo Money altera a mascara mudando um número da esquerda para a direita da virgula. Ex. Valor 55,44 após carregar um anexo fica 5,544 

### How to reproduce the bug

Dentro de qualquer formulário crie um campo Money e um FileUpload insira um valor dentro do campo Money e carregue um anexo dentro do FileUpload

### Package Version

3.0.0

### PHP Version

8.2.0

### Laravel Version

10.10

### Notes

O erro é apenas visual, ao salvar o formulário o valor correto é salvo.

**Comment by IgorDePaula** [comment]
Também estou com esse problema.



https://github.com/user-attachments/assets/85d7a05c-c628-44b7-af13-c2ee1d59fe18

**Comment by IgorDePaula** [comment]
@ghinozzi esse deu certo https://github.com/michaeld555/filament-money

**Comment by leandrocfe** [comment]
@IgorDePaula @ghinozzi @danielcei 

Verifiquem essa versao por favor
https://github.com/leandrocfe/filament-ptbr-form-fields/releases/tag/3.1.0
